src/portal/lib/src/vulnerability-scanning/result-bar-chart.component.ts

```diff
@@ -48,8 +48,11 @@ export class ResultBarChartComponent implements OnInit, OnDestroy {
     ) { }
 
     ngOnInit(): void {
-        if (this.tagStatus === "running") {
-            this.scanNow();
+        if (this.tagStatus === VULNERABILITY_SCAN_STATUS.running || this.tagStatus === VULNERABILITY_SCAN_STATUS.pending) {
+            // Avoid duplicated subscribing
+            this.stateCheckTimer = timer(0, STATE_CHECK_INTERVAL).subscribe(() => {
```

Comment:
should add "if (!this.stateCheckTimer)"

Reply:
Initialization is performed only once; its value does not exist.